docs/resources/grafana_dashboard.md

```diff
@@ -0,0 +1,64 @@
+---
+
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+
+page_title: "coralogix_grafana_dashboard Resource - terraform-provider-coralogix"
+subcategory: ""
+description: "Hosted grafana dashboard.
+* Official documentation https://grafana.com/docs/grafana/latest/dashboards/
+* HTTP API https://grafana.com/docs/grafana/latest/http_api/dashboard/"
+---
+
+# coralogix_grafana_dashboard (Resource)
+
+Hosted grafana dashboard.
+* [Official documentation](https://grafana.com/docs/grafana/latest/dashboards/)
+* [HTTP API](https://grafana.com/docs/grafana/latest/http_api/dashboard/)
+
+## Example Usage
+```hcl
+    resource "coralogix_grafana_dashboard" dashboard {
```

Comment:
This is different then what we discussed.
We wanted hosted_dashboard and that should be used for both grafana and opensearch

I would like us to have something like:
```
     resource "hosted_dashboard" dashboard {
       type = grafana
       config-json = ....
    }
```